client/src/Modules/AuthenticatedLanding/RegisteredEventListComponent.js

```diff
@@ -5,77 +5,58 @@ import React from 'react';
 import '../../Assets/scss/main.scss';
 import {API_URL} from '../../Utils/Urls';
 import EventCardComponent from '../Events/EventCardComponent';
-import {ajaxPost, ajaxGet, ajaxPut} from '../../Services/Http/Ajax';
-import {confirm, showMessage} from '../../Utils/Util';
-import {ToastContainer} from 'react-toastify';
-import {STATUS_ACTIVE} from '../../Utils/Constants';
+import axios from 'axios';
+import { formatDateDayAndDate } from '../../Utils/DateFormat';
+import { EventDateSorter, EventHandler } from '../../Utils/EventHandler';
 
 const RegisteredEventListComponent = (props) => {
-var isLoggedIn = true;
-var userId=1;
 let [registeredEventList,setRegisteredEvents] = React.useState([]);
  
 // Response will be received here and state is updated.
 React.useEffect(() => {
-        // Temporary list || 
-        const items = [
-            {
-                // date:'MONDAY, 4/2/2020',
-                day:[{
-                    id:'1',    // All other card variables come here
-                },{
-                    id:'2',
-                },,{
-                    id:'3',
-            }]}];
-            // --------------------------------------
-        setRegisteredEvents(items);
-        // let eventId = props.location.state.eventId;
-        let uri = API_URL.GET_REG_EVENT + `/${userId}`;
-        /*ajaxGet(uri, (response) => {
-            if (response.status === STATUS_ACTIVE) {               
-                setRegisteredEvents(response.content);
-            }
-            else {
-                console.log("Hello")
-                showMessage('error',response.message)
-            }
-        }) */
-
+/*Given a static zip code for now*/
+    try {
+        let zip_code = 43206;
+        const agencyUri = API_URL.EVENTS_LIST;
+        const resp =
+            axios.get(agencyUri, { params: { zip_code } })
+                .then(response => {
```

Comment:
I am confused with mixing a `.then` pattern with a  `try ... catch`. The then pattern has it's own catch function. I think it should be `async / await` if we're using `try ... catch` or just go with `.then() ... .catch()`.

Reply:
This will be rewritten by calling the webService function in ApiService.js file.